scalableminds/webknossos
# Fixed bug in removing layouts

This Quickfix disables a frontend bug, that appears when removing a layout in any tracing view mode.
### URL of deployed dev instance (used for testing):
- [https://deletinglayoutbug.webknossos.xyz](https://deletinglayoutbug.webknossos.xyz)

### Steps to test:
- open a tracing in any kind of view mode
- create a few new layouts
- remove them later => there should be no error in the console
- also selecting a layout should still work fine

### Issues:
- has non because I started fixing the bug right from the moment I found it.

------
